src/server/scripts/EasternKingdoms/Deadmines/instance_deadmines.cpp

```diff
@@ -87,13 +95,17 @@ class instance_deadmines : public InstanceMapScript
                 if (!pIronCladDoor)
                     return;
 
+                // Loads Mr. Smite's grid. We need this as he must be active in order for his texts to work.
+                instance->LoadGrid(MrSmite.GetPositionX(), MrSmite.GetPositionY());
```

Comment:
Thanks. I will move it to the current 2 positions where existing events for yells are.